### PR TITLE
fix: resolve scrollbar appearance on windows 🖱️

### LINKED ISSUE #5: Scrollbars show on everywhere on Windows
### Description

On Windows machines, it seems like scrollbars show everywhere in the Member Profile as well as the Admin Dashboard. It looks really ugly!

### Steps to Reproduce

1. Be on a Windows machine.
2. Go to Member Profile.
3. See scrollbars.

### Expected Behavior

The scrollbars should be hidden, and only appear once the scrolling happens (such as the default behavior on MacOS).

### Screenshots

Ugly Scrollbar (1:2)
Ugly Scrollbar (2:2)

### Additional Context

You need to be on a Windows machine to reproduce!

### PR DESCRIPTION
**Pull request by ayoaru**
## Description ✏️

Closes #5

- My first contribution of (hopefully) many! Added my name to the contributor list :)
- Resolved the scrollbar appearance. Usages of `overflow-scroll` needed to be modified to `overflow-auto`

## Type of Change 🐞

- [ ] Feature - A non-breaking change which adds functionality.
- [x] Fix - A non-breaking change which fixes an issue.
- [ ] Refactor - A change that neither fixes a bug nor adds a feature.
- [ ] Documentation - A change only to in-code or markdown documentation.
- [ ] Tests - A change that adds missing unit/integration tests.
- [ ] Chore - A change that is likely none of the above.

## Checklist ✅

- [x] I have done a self-review of my code.
- [x] I have manually tested my code (if applicable).
- [x] I have added/updated any relevant documentation (if applicable).

### CODE REVIEW

File: config/tailwind/tailwind.config.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import colors from 'tailwindcss/colors';
 
 /** @type {import('tailwindcss').Config} */
 export const tailwindConfig = {
+  blocklist: ['overflow-scroll'],
```
Comment (ramiAbdou) [inline config/tailwind/tailwind.config.js:5]: @ayoaru I added `overflow-scroll` to the blocklist to safeguard us from using it in the future!

**Review by ramiAbdou — APPROVED** [review]
Thank you so much, @ayoaru! This bug being fixed is such a huge relief! 😭

**Comment by ayoaru** [comment]
@ramiAbdou No worries! Happy to help 💫

File: config/tailwind/tailwind.config.js (hunk 1 above)
Comment (ayoaru) [reply config/tailwind/tailwind.config.js:5]: Perfect, do I need to do anything else on my end or am I good to merge?

**Pull request merged**